app/js/components/patientDashboard/View.jsx

```diff
@@ -2,43 +2,50 @@ import React from 'react';
 import dateFns, { distanceInWordsToNow } from 'date-fns';
 import PropTypes from 'prop-types';
 
-const View = props => (
+const View = ({
+  patient: {
+    person, patientId, attributes, patientIdentifier,
+  }, toggleDetailsView, showContactInfo, note,
+}) => (
   <div className="patient-header ">
     <div className="demographics">
       <h1 className="name">
         <span>
-          <span className="PersonName-givenName">{props.patient.person.personName.givenName}&nbsp;&nbsp;</span>
+          <span className="PersonName-givenName">{person.personName.givenName}&nbsp;&nbsp;</span>
           <em>Given</em>
```

Comment:
I'd think this should be internationalized

Reply:
@wluyima When you say internationalized, do you mean language localisation, please provide more context. Thanks

Reply:
Yes

Reply:
okay, will take this into consideration, but for now, we're more focused on completing the functionalities that the current order entry ui has, and to our knowledge it currently doesn't hold  implementation to support language internationalisation. So will make sure to discuss this with the team and hopefully implement if time permits. Thank you

Reply:
In particular, we haven't yet decided (or done much research) into a what framework/tool we will use to localize React Apps (see https://talk.openmrs.org/t/owas-localization/17750/6).

This is something we will need to do sooner rather that later... but I don't think it should block a pull request like this being merged in... there should likely be a separate ticket about setting up localization, and once it gets set up, a ticket for going back and localizing all text based on whatever framework we set up.